docs/core/about.md

```diff
@@ -5,47 +5,45 @@ ms.date: 09/17/2019
-- **Compatible:** .NET Core is compatible with .NET Framework, Xamarin, and Mono, via [.NET Standard](../standard/net-standard.md).
-- **Open source:** The .NET Core platform is open source, using MIT and Apache 2 licenses. .NET Core is a [.NET Foundation](https://dotnetfoundation.org/) project.
-- **Supported by Microsoft:** .NET Core is supported by Microsoft, per [.NET Core Support](https://dotnet.microsoft.com/platform/support/policy).
+- **Open source:** The .NET Core platform is [open source](https://github.com/dotnet/core), using MIT and Apache 2 licenses. .NET Core is a [.NET Foundation](https://dotnetfoundation.org/) project.
+- **Modern:** It implements modern paradigms like asyncronous programming, no-copy patterns using structs, and resource governance for containers.
+- **Performance:**  Delivers [high performance](https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-core-3-0/) with features like [hardware intrinsics](https://devblogs.microsoft.com/dotnet/hardware-intrinsics-in-net-core/), [tiered compilation](https://github.com/dotnet/coreclr/blob/master/Documentation/design-docs/tiered-compilation.md), and [Span<T>](../standard/memory-and-spans/index.md).
+- **Consistent across environments:** Runs your code with the same behavior on multiple OSes and architectures, including x64, x86, and ARM.
+- **Command-line tools:**  Includes easy-to-use command-line tools that can be used for local development and for continuous-integration.
+- **Flexible deployment:** You can include .NET Core in your app or install it side-by-side (user-wide or system-wide installations). Can be used with [Docker containers](docker/introduction.md).
 
 ## Languages
 
 C#, Visual Basic, and F# languages can be used to write applications and libraries for .NET Core. These languages can be used in your favorite text editor or Integrated Development Environment (IDE), including:
 
 - [Visual Studio](https://visualstudio.microsoft.com/vs/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link)
 - [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
-- Sublime Text
-- Vim
 
-This integration is provided, in part, by the contributors of the [OmniSharp](https://www.omnisharp.net/) and [Ionide](http://ionide.io) projects.
+Editor integration is provided, in part, by the contributors of the [OmniSharp](https://www.omnisharp.net/) and [Ionide](http://ionide.io) projects.
 
 ## APIs
 
-.NET Core exposes APIs for many scenarios, a few of which follow:
+.NET Core exposes frameworks for building any kind of app:
+
+* [Cloud apps, with ASP.NET Core](/aspnet/core/)
+* [Mobile apps, with Xamarin](/xamarin)
+* [IoT apps, with System.Device.GPIO](/archive/msdn-magazine/2019/august/net-core-cross-platform-iot-programming-with-net-core-3-0)
```

Comment:
for links in the archive, I'd use the full link. we only use the cross repo for things that would be in the VS offline books

```suggestion
* IoT apps with [System.Device.GPIO](https://docs.microsoft.com/archive/msdn-magazine/2019/august/net-core-cross-platform-iot-programming-with-net-core-3-0)
```